<launch>
    <node pkg="automow_planning" type="path_planner.py" name="path_planner" output="screen" required="true">
        <param name="field_csv_file" type="string" value="$(find automow_maps)/maps/survey_solarhouse_may30.csv" />
        <param name="goal_timeout" value="60.0" />
    </node>
</launch>
